---
layout: default
title: "Igor S. Pessi, PhD: Research"
permalink: /research/
---

<div class="wrapper margin-bottom">
  <h2><i class="fas fa-dna"></i>Research</h2>
  <!-- <p>My research is focused on the study of microbial communities in polar environments.</p>
  <!-- FROM: 3-year grant apll form.docx -->
  <!-- <p>Since my master’s studies my research has focused on microbial communities in the Arctic and the Antarctic and covered several different aspects of their ecology. My master’s studies involved the study of the functional and genetic diversity of microbial communities in soil, water and snow samples from Maritime Antarctica using culture-dependent and independent approaches. During my PhD in Belgium, I optimized the high-throughput sequencing methodology aiming at a particular group of microorganisms, the cyanobacteria. I then applied this approach to investigate patterns of cyanobacterial distribution in a range of polar environments.</p>
  <!-- FROM: ISP academy 2018 final.docx -->
  <!-- <p class="end-paragraph">My master’s and PhD researches focused on microbial communities in the Arctic and the Antarctic and covered several different aspects of their ecology. During my master’s studies in Brazil, I investigated the functional and genetic diversity of microbial communities in soil, water and snow samples from Maritime Antarctica using culture-dependent and HTS approaches. During my PhD in Belgium, I optimized the HTS methodology aiming at a particular group of microorganisms, the cyanobacteria. I then applied this approach to investigate patterns of cyanobacterial distribution in a range of polar environments, including (i) Antarctic lakes encompassing a range of environmental conditions and biogeographic provinces; (ii) biological soil crusts (BSCs) from nunataks and mountain ranges in East Antarctica; (iii) BSCs along a successional gradient in several glacier forefields in Svalbard and Maritime Antarctica; and (iv) BSCs at different stages of development in Svalbard.</p> -->
</div>

<!-- <div class="wrapper margin-bottom">

  <div class="slideshow">
    <img src="../images/slideshow1.jpg" style="width:100%">
  </div>

  <div class="slideshow">
    <img src="../images/slideshow2.jpg" style="width:100%">
  </div>

  <div class="slideshow">
    <img src="../images/slideshow3.jpg" style="width:100%">
  </div>

  <a class="slideshow-prev" onclick="plusSlides(-1)">&#10094;</a>
  <a class="slideshow-next" onclick="plusSlides(1)">&#10095;</a>

  <div class="slideshow-caption-wrapper">
    <p id="slideshow-caption"></p>
  </div>

  <div class="slideshow-row">

    <div class="slideshow-column">
      <img class="slideshow-thumb " src="../images/slideshow1.jpg" style="width:100%" onclick="currentSlide(1)" alt="Kilpisjärvi, Finland">
    </div>

    <div class="slideshow-column">
      <img class="slideshow-thumb " src="../images/slideshow2.jpg" style="width:100%" onclick="currentSlide(2)" alt="Kilpisjärvi, Finland">
    </div>

    <div class="slideshow-column">
      <img class="slideshow-thumb " src="../images/slideshow3.jpg" style="width:100%" onclick="currentSlide(3)" alt="Svalbard, Norway">
    </div>

  </div>

</div> -->

<script>
  var slideIndex = 1;
  showSlides(slideIndex);

  function plusSlides(n) {
    showSlides(slideIndex += n);
  }

  function currentSlide(n) {
    showSlides(slideIndex = n);
  }

  function showSlides(n) {
    var i;
    var slides = document.getElementsByClassName("slideshow");
    var dots = document.getElementsByClassName("slideshow-thumb");
    var captionText = document.getElementById("slideshow-caption");
    if (n > slides.length) {
      slideIndex = 1
    }
    if (n < 1) {
      slideIndex = slides.length
    }
    for (i = 0; i < slides.length; i++) {
      slides[i].style.display = "none";
    }
    for (i = 0; i < dots.length; i++) {
      dots[i].className = dots[i].className.replace(" slideshow-thumb-active", "");
    }
    slides[slideIndex-1].style.display = "block";
    dots[slideIndex-1].className += " slideshow-thumb-active";
    captionText.innerHTML = dots[slideIndex-1].alt;
  }
</script>

<!-- <div class="wrapper">
  <h2><i class="fas fa-dna"></i>Description of past research</h2>
  <!-- FROM: application Hultman postdoc IGOR PESSI.pdf -->
  <!-- <p>I have been working with microbiology since my undergraduate studies at UFRGS (Porto Alegre, Brazil), where I was involved with several projects dealing with the study of the physiology and molecular biology of the fungus Bipolaris sorokiniana, a wheat pathogen of great importance for Brazilian agriculture (Poloni et al., 2008, 2009a, 2009b). My involvement with Antarctic microbial ecology began in 2010 when I started my master’s studies, also at UFRGS. During this time, I investigated the functional and genetic diversity of microbial communities in soil, water, and snow samples from King George Island, Antarctic Peninsula using culture-dependent and high-throughput sequencing (HTS) approaches (Pessi et al., 2012, 2015).</p>

  <p>I started my PhD studies at the University of Liège (Belgium) in 2012. My PhD project involved the application of the HTS approach to the study of cyanobacterial diversity in a range of Arctic and Antarctic environments. During the first year, I performed a pilot study in order to optimize the 454 pyrosequencing methodology for the study of cyanobacterial communities, which involved, among others, the evaluation of a variety of bioinformatics tools for the analysis of high-throughput amplicon sequencing data (Pessi et al., 2016). This optimized protocol served as the experimental basis for other studies carried out during my PhD thesis, in which HTS of 16S rRNA gene sequences was used to investigate patterns of cyanobacterial distribution in a range of polar environments, including (i) Antarctic lakes encompassing a range of environmental conditions and biogeographic provinces (Pessi et al., submit.); (ii) biological soil crusts (BSCs) from nunataks and mountain ranges in the Sør Rondane Mountains, Dronning Maud Land (Pushkareva et al., submit.); (iii) BSCs along a successional gradient in several glacier forefields in Petunia Bay, Svalbard (Pessi et al., submit.) and Fildes Peninsula, King George Island; and (iv) BSCs at different stages of development in Petunia Bay (Pushkareva et al., 2015). Moreover, metagenomic data from several Antarctic lakes and BSCs from glacier forefields in Petunia Bay have been recently obtained and are currently being analysed. I was also involved with the sequencing of the genome of Phormidesmis priestleyi ULC007, a cyanobacterium isolated from Lake Bruehwiler, Larsemann Hills, Antarctica (Lara et al., 2017). Finally, my PhD studies also included a four-month research stay at UNIS (Longyearbyen, Svalbard), where I applied the HTS approach to investigate the response of soil microbial communities to increased snow cover. During this period, I also acquired valuable fieldwork experience in the Arctic.</p>

  <p class="end-paragraph">In summary, I have an extensive experience in molecular ecology investigations of polar microbial communities. This includes sampling, sample processing (DNA extraction, PCR/qPCR, and library preparation), and bioinformatics, phylogenetic, and statistical analyses. I have a good knowledge of the common bioinformatics tools for the analysis of HTS data (including amplicon sequencing, genomic, and metagenomic data), as well as UNIX and R.</p>
</div> -->
